crates/sui-graphql-e2e-tests/tests/consistency/coins.exp

```diff
@@ -1,4 +1,4 @@
-processed 21 tasks
```

Comment:
the previous snapshot test was overly complex and did not actually test for behavior with `objects_snapshot` updating. To follow along, check out comments on `coins.move`